MedicalApp.BuisnessServices/Mappers/Implementation/PatientMapper.cs

```diff
@@ -8,8 +8,15 @@
 
 namespace MedicalApp.BuisnessServices.Mappers.Implementation
 {
-    class PatientMapper:IMapper<Patient, PatientEntity>
+    class PatientMapper : IMapper<Patient, PatientEntity>
     {
+        public PatientMapper()
+        {
+            roomMapper = new RoomMapper();
+        }
+
+        private readonly RoomMapper roomMapper;
+
         public void MapToEntity(Patient item, PatientEntity targetItem)
```

Comment:
Minor suggestion:
provide more informative name

Reply:
The variables was renamed